e2e/cypress/integration/integrations/plugin_slash_command_stays_visible_spec.js

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @messaging
+
+describe('Messaging', () => {
```

Comment:
The Group should be `@integrations` and instead of `Messaging` the describe block should be `Integrations`

Reply:
Ah my mistake. Thank-you for pointing that out.